src/main/java/racingcar/Car.java

```diff
@@ -8,5 +10,18 @@ public Car(String name) {
         this.name = name;
     }
 
-    // 추가 기능 구현
+    public void moveForward() {
+        int randomValue = Randoms.pickNumberInRange(0,9);
+        if (randomValue >= 4) {
+            position++;
+        }
```

Comment:
이렇게 바꾸기 vs 그냥 쓰기
각각 어떤 장단점이 있을까요?

```suggestion
        if (Randoms.pickNumberInRange(0,9) >= 4) {
            position++;
        }
```

Reply:
바꿨을 때의 장단점
장점:
코드가 간결해집니다. 한 번 사용되는 간단한 로직일 경우에, 중복 코드를 최소화할 수 있습니다.
단점:
매번 Randoms.pickNumberInRange를 호출하기 때문에 성능상 단점이 있을 수 있습니다.
코드 한 줄에 많은 동작이 있다면 가독성이 감소할 수 있습니다. 복잡하고 다양한 동작을 수행할 경우에 이해하기 어려워질 수 있습니다.
이어져 코드가 복잡해지면 유지보수가 어려워질 수 있습니다.

randomValue 변수를 사용한다면 코드를 읽기가 더 쉽습니다. 다른 부분에서 동일한 난수를 사용한다면 재사용에 용이합니다. 또한 난수 생성 로직이 변경되더라도 해당 부분만 수정하면 됩니다.

Reply:
정리 깔끔하네요! 그런데 매번 호출하는건 위 아래 전부 동일에서 성능은 사실상 동일합니다!
현재 코드에선 각각 가독성, 코드 길이에서 우위가 있겠네요!